en/docs/learn/analytics/white-labeling-tenants.md

```diff
@@ -0,0 +1,23 @@
+# White Labeling for Tenants
+This section explains how to white label analytics dashboard for tenants.
+1. Add following configuration in the deployment.yaml
+    ```yaml
+    themeConfigProviderClass: org.wso2.analytics.apim.dashboards.theme.config.provider.CustomDashboardThemeConfigProvider
+    logoFileName: logo.png
+    faviconFileName: favicon.ico
```

Comment:
Shouldn't we mention that these file names are configurable. i.e. No need of having the same file name as it is.